modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/storage/ItRebalanceDistributedTest.java

```diff
@@ -1076,29 +1069,41 @@ private static void createZone(Node node, String zoneName, int partitions, int r
     }
 
     private static void createZone(Node node, String zoneName, int partitions, int replicas, boolean testDataStorage) {
-        DistributionZonesTestUtil.createZone(
-                node.distributionZoneManager,
+        DistributionZonesTestUtil.createZoneWithDataStorage(
+                node.catalogManager,
                 zoneName,
                 partitions,
                 replicas,
-                testDataStorage ? (dataStorageChange -> dataStorageChange.convert(TestDataStorageChange.class)) : null
+                testDataStorage ? TestStorageEngine.ENGINE_NAME : null
         );
     }
 
     private static void alterZone(Node node, String zoneName, int replicas) {
-        alterZoneReplicas(node.distributionZoneManager, zoneName, replicas);
+        DistributionZonesTestUtil.alterZone(node.catalogManager, zoneName, replicas);
     }
 
     private static void createTable(Node node, String zoneName, String tableName) {
-        TableDefinition schTbl1 = SchemaBuilders.tableBuilder(DEFAULT_SCHEMA_NAME, tableName).columns(
+        TableTestUtils.createTable(
+                node.catalogManager,
+                DEFAULT_SCHEMA_NAME,
+                zoneName,
+                tableName,
+                List.of(
+                        ColumnParams.builder().name("key").type(org.apache.ignite.sql.ColumnType.INT64).build(),
+                        ColumnParams.builder().name("val").type(org.apache.ignite.sql.ColumnType.INT32).nullable(true).build()
+                ),
+                List.of("key")
+        );
+
+        TableDefinition tableDefinition = SchemaBuilders.tableBuilder(DEFAULT_SCHEMA_NAME, tableName).columns(
```

Comment:
Why do you use Utility class to create a table instead of using CatalogManager directly?

Reply:
For convenience, when I started doing the task, I discovered that many tests created the zone directly through the manager instead of using auxiliary methods, which introduced quite a lot of unpleasant refactorings.

To reduce refactoring in the future, I created this method.

Reply:
I don't understand how this approach could help if smth will be changed in future.
Having a shortcut method for building CreateTableParams only looks ok.
But, let's call CatalogManager explicitly.

```suggestion
        CreateTableParams params = TableTestUtils.createTable(         
                DEFAULT_SCHEMA_NAME,
                zoneName,
                tableName,
                List.of(
                        ColumnParams.builder().name("key").type(org.apache.ignite.sql.ColumnType.INT64).build(),
                        ColumnParams.builder().name("val").type(org.apache.ignite.sql.ColumnType.INT32).nullable(true).build()
                ),
                List.of("key")
        );
        node.catalogManager().createTable(params);
      
        TableDefinition tableDefinition = SchemaBuilders.tableBuilder(DEFAULT_SCHEMA_NAME, tableName).columns(
```

Reply:
It took me quite a bit of effort to fix the tests and change them to the new API.
I was helped in this by using methods instead of explicitly calling the API.
Since this method has proven to be more effective so far (it will also reduce the pain of trying to merge `catalog-feature` into `main`), I will stick with it and try to stick with it.